project-api/project-api-test/src/test/resources/myproject_with_lang_import/util/file-util.bal

```diff
@@ -0,0 +1,2 @@
+public function foo() {
+}
```

Comment:
Let's add a new line at the end of the file

Reply:
added in the 274babad95104c166f871220329483954e88cf7b commit